### ISSUE TITLE
plugin does not detect failing test

### ISSUE THREAD

**Issue by ghost** [kotest/kotest#5538]
there seems to be an issue when using strikt.io assertions with kotlintest:

```
import io.kotlintest.specs.DescribeSpec
import strikt.api.expectThat
import strikt.assertions.contains

class FooTest : DescribeSpec() {

    init {
        describe("describe") {
            it("should fail") {
                expectThat(listOf("foo")).contains("bar")
            }
        }
    }
}
```

this test fails if run within gradle. However when i run it from IntelliJ the testrunner does not seem to properly detect the failed teststatus:
* test runner exits with status 0
* test spinner keeps spinning (see screenshot)

plugin version: v3.3.0.11
intellij version: IntelliJ IDEA 2019.1.3 (Ultimate Edition)
Build #IU-191.7479.19, built on May 28, 2019

![image](https://user-images.githubusercontent.com/51798098/61182071-831e0580-a62e-11e9-943e-b3f28db2ed5f.png)

**Comment by sksamuel** [comment]
Idea expects that standard output is in a specific format for it to work with external processors.
The format is `##teamcity[$command...]`
Looks like strikt uses the [ ] itself for values.
I can add a \ before [ and ] in the output but I'm not sure if that will actually do anything. It might be the case that strikt needs to pick another character.

**Comment by sksamuel** [comment]
If I do a snapshot build of KotlinTest can you check this? You would just need to update the version in your project.

**Comment by ghost** [comment]
@sksamuel sure i would do that. do you mean a new version of kotlintest or the intellij plugin?

**Comment by sksamuel** [comment]
KotlinTest.

**Comment by sksamuel** [comment]
This will be fixed now if you try 4.1.1.
